synced-files/.github/workflows/configlet.yml

```diff
@@ -0,0 +1,18 @@
+name: Configlet
+
+on: [push, pull_request, workflow_dispatch]
+
+jobs:
+  configlet:
+    name: Lint
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f # 2.3.4
```

Comment:
The previous version of the workflow did not have the `# 2.3.4` comment added.

Reply:
FYI that AFAIK Dependabot will not update version comments and they can then fall behind (quite quickly in some cases), which then turn them to bad information.